app/controllers/tickets_controller.rb

```diff
@@ -66,10 +80,39 @@ def find
   end
 
   def create_params
-    params.require(:data).permit(:title, :worker_id, :description, :state)
+    params.require(:data).permit(:title, :worker_id, :description)
   end
 
   def update_params
     params.require(:data).permit(:title, :description)
   end
+
+  def state(state_method, can_change_state)
```

Comment:
`def state` sound like a reader, name methods to make them look self explanatory. In this case what we do? We update ticket state, so at leas this should look like `def udate_state`